content/posts/see_presentation.Rmd

```diff
@@ -21,36 +21,67 @@ include_graphics("https://github.com/easystats/see/raw/master/man/figures/logo.p
 
 
 
-## see
+## The see package
 
-[We](https://github.com/orgs/easystats/people) have recently decided to collaborate around the new [**easystats project**](https://github.com/easystats), a set of packages designed to make your life *easier*. This project encompasses several packages, devoted for instance to [model access](https://github.com/easystats/insight) or [Bayesian analysis](https://github.com/easystats/bayestestR), [indices of model performance](https://github.com/easystats/performance) or  [visualisation](https://github.com/easystats/see).
+[We](https://github.com/orgs/easystats/people) have recently decided to collaborate around the new [**easystats project**](https://github.com/easystats), a set of packages designed to make your life easier. This project encompasses several packages, devoted for instance to [model access](https://github.com/easystats/insight) or [Bayesian analysis](https://github.com/easystats/bayestestR), [indices of model performance](https://github.com/easystats/performance) or [visualisation](https://github.com/easystats/see).
 
-***Without further ado, please let us introduce the latest addition to the easyverse; the ***  [**`see`**](https://github.com/easystats/see) ***package!***
+***Without further ado, please let us introduce the latest addition to the easyverse; the see ackage!***
```

Comment:
Maybe: "***Without further ado, we introduce the latest addition to the easyverse: the see package!***"

A couple light grammatical things and streamlined language a bit. Feel free to keep or discard, of course